src/Editor/main.cpp

```diff
@@ -17,6 +17,10 @@
 #include "ImGui/OpenGLImplementation.hpp"
 #include <imgui.h>
 
+#ifdef VR_SUPPORT
```

Comment:
One of our requirements is that we should be able to choose VR/non-VR at launch time. It might be useful to keep doors open to enable VR at runtime instead of having to precompile with VR support.